### ISSUE TITLE
Differentiate system messages from chat messages in the chat column

### ISSUE THREAD

**Issue by superemily** [dasevilla/rdio-party#122]
When someone hearts or thumbs up something, the message currently goes to chat and it looks identical to an actual chat message. Let's some how indicate in there that it's a system message (different background or icon associated with that message?)

**Comment by MikeNGarrett** [comment]
I was trying to imagine how you could handle events separate from the chat. Here's a quick mock I built that pulls out events into its own column. 
![events-tracking](https://cloud.githubusercontent.com/assets/394888/7618254/d0321c6e-f97e-11e4-9da2-689c4fb2783d.jpg)

**Comment by rstecker** [comment]
Most folks are adverse to adding more columns (myself included) so I think the suggestion is turned down.

I like the idea of using icons though to replace the text to make it clear it's not just a chat msg & to keep it concise.

Gabe shared us a mock for what his mobile chat looks like (in the yet-to-launch iOS version) and we all liked it in that it kept the actions in the chat but differentiated them.

![ios-rdio-party-chat](https://cloud.githubusercontent.com/assets/3209024/7637858/c349d8d2-fa25-11e4-85a1-02484216df2d.png)

**Comment by MikeNGarrett** [comment]
Not a problem. I understand adding another column would be a bit difficult. 

I was trying to address having too many items in the chat list that weren't related to chat. It's going to be difficult as this app gets more popular to differentiate between chat messages and events such as likes, skips, favorites, etc.

What do you think?

**Comment by dasevilla** [comment]
Perhaps multiple system messages in a row could roll-up (i.e. 3 people have favorited this)

**Comment by BCWilsonDotCom** [comment]
Yeah we're definitely going to need some way to differentiate chat messages from site interactions, etc. I think it's really just a case of highlighting and displaying non-chat items differently.

**Comment by MikeNGarrett** [comment]
I like the idea of rolling them up. That could certainly help, but that would only work with the same types of items. You still have a variety of different event types. 

As for displaying events differently, the biggest premium is on vertical space in the chat interface. How can we reduce that as much as possible?

**Comment by rstecker** [comment]
Ah, I think we see the problem--
> the biggest premium is on vertical space in the chat interface
But why?  Live in the moment! The past is fleeting, let it scroll away.  I think that's the actual discussion/design point to talk about.

**Comment by BCWilsonDotCom** [comment]
I'm not big on the idea of rolling them up actually. If the chat is super active and then actions such as favorites, likes, dislikes, etc get rolled up, they may not even be seen.

**Comment by jackdbernier** [comment]
Or aggregating some events together just like slack does.

```
[user_x] favorited [track_y]. Also [user_a] vote to skip.
```

**Comment by MikeNGarrett** [comment]
Haha. Sure, sure. The present is the best. Maybe I'm jumping the gun. I'm trying to think of the chat experience when there are more than 15 people in the party. It can already get crazy when there are less than 15. How can you follow a discussion when there are 100 people all causing events and all that information appears in the chat?

**Comment by MikeNGarrett** [comment]
I have a couple of proposals to start reducing the number of events in chat. 

1) Add like/dislike counts to the buttons.
https://jsfiddle.net/MikeNGarrett/vmdxcjk7/4/embedded/result/
2) Add favorites to user profile images.
![favs](https://cloud.githubusercontent.com/assets/394888/7667784/c388d764-fbe5-11e4-93d2-7f15172c4940.jpg)
3) Reduce track change size. 
![smaller-events](https://cloud.githubusercontent.com/assets/394888/7667837/c22f883e-fbe7-11e4-9d48-16cdae171ac2.jpg)

Let me know if this is too much.

**Comment by superemily** [comment]
That sounds fine, just as long as we can capture the excitement of repeatedly hitting the favorite and unfavorite button over and over again somehow. If not chat spam, I guess  a throbbing heart next to your user icon would do the trick!

**Comment by jackdbernier** [comment]
I like the idea of smaller events in the chat. 

I'm not sure about the favorite icon on the user profile. 1) because we can't see them all at once. 2) because once we change track we loose the history.

Let me suggest something. What if we would stick a small profile image of the ones liking the track event in the chat. I really like the having the events in one place. Skip, Favorite and so on.

But that's me.

**Comment by BCWilsonDotCom** [comment]
+1 on the reduction of the track change size. That looks great.

**Comment by MikeNGarrett** [comment]
@superemily +1 for excitement factor. 

@jackdbernier I have an idea for (1) that I'll propose later and (2) do we need favorite history? If we do need an event history, could we concatenate it into the track listing in chat? eg Track by artist - 2 likes, 5 skips, 2 favorites.

**Comment by gabek** [comment]
I like the icon on the user images.  It's a bit of a throwback to Turntable.fm where the avatar would bob its head when the user voted up a track.
